model/src/w3initmd.F90

```diff
@@ -410,6 +410,7 @@ SUBROUTINE W3INIT ( IMOD, IsMulti, FEXT, MDS, MTRACE, ODAT, FLGRD,  FLGR2, FLGD,
          UA, UD, U10, U10D, AS
 #ifdef W3_MPI
     USE W3ADATMD, ONLY: MPI_COMM_WAVE, MPI_COMM_WCMP
+    USE MPICOMM
```

Comment:
For all of these USE MPICOM, if we can add the "only" statement that would be nice.  I know it's not uniformly done in WW3 right now, but since this is an addition, it would be good to add it.

Reply:
I'll push a commit to make these all have an only statement